wpilibj/src/main/java/edu/wpi/first/wpilibj/livewindow/LiveWindow.java

```diff
@@ -64,17 +63,13 @@ public static synchronized void setEnabled(boolean enabled) {
       startLiveWindow = enabled;
       liveWindowEnabled = enabled;
       updateValues(); // Force table generation now to make sure everything is defined
-      Scheduler scheduler = Scheduler.getInstance();
       if (enabled) {
         System.out.println("Starting live window mode.");
-        scheduler.disable();
-        scheduler.removeAll();
```

Comment:
Do we still want commands to be unscheduled when starting LW mode?

Reply:
I realized this was actually broken because of new commands, so we need to rethink how we want to do this. Since it was broken anyway.

Reply:
Seems like LiveWindow should have a way to add callbacks for starting/stopping LW mode. Then when the Schedulers are used, they can add the relevant callbacks.

Reply:
Agreed. I would likely do that in a separate PR because otherwise this one becomes much harder to review.